sherpa/cpp_api/grpc/online-grpc-client.cc

```diff
@@ -0,0 +1,152 @@
+// sherpa/cpp_api/grpc/online-grpc-client.cc
+
+#include <chrono>  // NOLINT
+#include <random>
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <cstdlib>
+#include <iostream>
+
+#include "grpc/grpc.h"
+#include "grpcpp/channel.h"
+#include "grpcpp/client_context.h"
+#include "grpcpp/create_channel.h"
+
+#include "sherpa/csrc/wav.h"
+#include "sherpa/csrc/log.h"
+#include "sherpa/cpp_api/parse-options.h"
+#include "online-grpc-client-impl.h"
+
+static constexpr const char *kUsageMessage = R"(
+Automatic speech recognition with sherpa using grpc.
+
+Usage:
+
+sherpa-online-grpc-client --help
+
+sherpa-online-grpc-client \
+  --server-ip=127.0.0.1 \
+  --server-port=6006 \
+  --wav-path=/path/to/foo.wav
+
+or
+
+sherpa-online-grpc-client \
+  --server-ip=127.0.0.1 \
+  --server-port=6006 \
+  --wav-scp=/path/to/wav.scp
+)";
+
+using namespace std;
```

Comment:
Please don't use `using namespace`.

Please use `std::vector` and `std::string`.